lib/sqlalchemy/orm/query.py

```diff
@@ -3169,7 +3177,9 @@ def delete(
 
         """
 
-        bulk_del = BulkDelete(self)
+        delete_args = delete_args or {}
```

Comment:
bulk delete accepts none, so let's avoid creating a dict here, and just pass none to it